test/integration/features/basic.feature

```diff
@@ -1,8 +1,31 @@
 @basic
 Feature: Basic
-  In order to use Minishift
-  As a user
-  I need to be able to bring up a test environment
+  As a user I can perform basic operations of Minishift and OpenShift
+
+  Scenario: User can install default add-ons
+    Given Minishift has state "Does Not Exist"
+    When executing "minishift addons install --defaults" succeeds
+    Then stdout should contain
+     """
+     Default add-ons anyuid, admin-user, xpaas installed
+     """
+
+  Scenario: User can enable the anyuid add-on
+    Given Minishift has state "Does Not Exist"
+    When executing "minishift addons enable anyuid" succeeds
+    Then stdout should contain
+     """
+     Addon 'anyuid' enabled
+     """
+
+  Scenario: User can list enabled plugins
+    Given Minishift has state "Does Not Exist"
+    When executing "minishift addons list" succeeds
+    Then stdout should contain
+     """
+     - anyuid         : enabled    P(0)
+     - admin-user     : disabled   P(0)
+     """
```

Comment:
I think we need to test test with add-ons and without add-ons. If we are testing only with add-ons than we might miss issues that might exist without add-ons . WDYT?